stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_composer.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+
+    <androidx.appcompat.widget.LinearLayoutCompat
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        >
+
+        <FrameLayout
+            android:id="@+id/leadingContent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            app:layout_constraintRight_toRightOf="@+id/centerContent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
+        <FrameLayout
+            android:id="@+id/centerContent"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_weight="1"
+            app:layout_constraintLeft_toRightOf="@+id/leadingContent"
+            app:layout_constraintRight_toLeftOf="@+id/trailingContent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
+        <FrameLayout
+            android:id="@+id/trailingContent"
```

Comment:
What happens if the Center content is too big? Because of `wrap_content` it could fall outside of bounds, or push the trailing content out of the screen. Sme goes for leading content

Reply:
Currently the `centerContent` is configured to take all the remaining horizontal space in the container. IMO, this should be adjusted so that it also has minimal width and height set. Also, we should configure leading and trailing containers to have minimal sizes too. Not sure how to do it properly.

Reply:
We could always use weights or something like that, constraints alternatively. Since the Composer is a Constraint layout, we can use proper constraints and constraint chain attributes to make it work as it's intended.